torch/csrc/jit/python/script_init.cpp

```diff
@@ -783,11 +787,51 @@ void initJitScriptBindings(PyObject* module) {
                   if (auto method = self.find_method(name)) {
                     return py::cast(*method);
                   }
+                  if (self.has_property(name)) {
+                    auto prop = self.get_property(name);
+                    auto getter_func = py::cast(prop.getter_func);
+                    auto setter_func = py::cast(prop.setter_func);
+                    return getter_func();
+                  }
                   return toPyObject(self.attr(name));
                 } catch (const ObjectAttributeError& err) {
                   throw AttributeError("%s", err.what());
                 }
               })
+          .def(
+              "__setattr__",
+              [](Object& self, const std::string& name, py::object value) {
+                try {
+                  if (self.has_property(name)) {
+                    auto prop = self.get_property(name);
+                    auto getter_func = py::cast(prop.getter_func);
+                    if (!prop.setter_func.has_value()) {
+                      TORCH_CHECK(
+                          false,
+                          "Can't set value to '",
+                          name,
+                          "' because setter function is not given.");
```

Comment:
Let's find out what happens in regular Python when you try to use the setter of a property that doesn't have one and let's try to copy that behaviour as much as possible.

Reply:
cool. will verify

Reply:
It just says "can't set attribute", so I updated our error message so that it is same as python